src/main/java/org/folio/inventory/consortium/util/InstanceOperationsHelper.java

```diff
@@ -0,0 +1,95 @@
+package org.folio.inventory.consortium.util;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import org.apache.http.HttpStatus;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.inventory.consortium.handlers.TenantProvider;
+import org.folio.inventory.dataimport.exceptions.OptimisticLockingException;
+import org.folio.inventory.domain.instances.Instance;
+import org.folio.inventory.exceptions.NotFoundException;
+import org.folio.kafka.exception.DuplicateEventException;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang.StringUtils.isNotBlank;
+import static org.folio.inventory.dataimport.util.DataImportConstants.UNIQUE_ID_ERROR_MESSAGE;
+
+public class InstanceOperationsHelper {
+
+  private static final Logger LOGGER = LogManager.getLogger(InstanceOperationsHelper.class);
+
+  public Future<Instance> addInstance(Instance instance, TenantProvider tenantProvider) {
+
+    LOGGER.info("addInstance :: Publishing instance with InstanceId={} to tenant={}", instance.getId(),
```

Comment:
nit: This class does not have a context of "publishing" but it is "adding". I think the log message should be "adding instance" instead of "publishing instance"

Reply:
thanks, completed